docs/UserGuide.md

```diff
@@ -55,14 +55,11 @@ Here's a quick summary of the available sections in the user guide:
 * Parameters can be in any order.<br>
   e.g. if the command specifies `ans/ ANSWER opt/ OPTION1`, `opt/ OPTION1 ans/ ANSWER` is also acceptable.
 
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `ans/t ans/abc`, only `ans/abc` will be taken.
-
```

Comment:
Another reason I say this is because it will be very repetitive to repeat this whenever this rule is applicable (e.g. this rule holds  not just for ans/, but for qn/, i/ etc. etc.)

Reply:
I agree that this shouldn't be deleted, even though it's a bit weird to be in the user guide. This was actually an issue pointed out in previous years PE, so we should leave it there to prevent loss of marks during PE.

Reply:
This is a functionality of just `MCQ` and `TFQ`, not `SAQ`. (This referring to taking `ans/abc` when multiple `ans/` are given)
In SAQ if I have 2 `ans/` parameters, an error is thrown.

So, I shifted these lines to MCQ and TFQ descriptions. I think it could be cited as a possible bug in the actual PE, which is why I shifted.

Reply:
Hmm, I think the behaviour is consistent for SAQ too. Try ```saq qn/a ans/a ans/k/abc i/2```. I believe I take the last instance of ans/ in addsaqparser as well.  I think it is safer to add this in and there will be fewer bugs reported than if we take it out and try to clarify every instance where this is true

Reply:
My bad. The command I tested with had wrong format. Will change in the next commit.